Sources/OTLPCore/Metrics/OTelMetricDataModel+Proto.swift

```diff
@@ -87,7 +80,6 @@ extension Opentelemetry_Proto_Metrics_V1_Sum {
     }
 }
 
-@_spi(Metrics)
 extension Opentelemetry_Proto_Metrics_V1_AggregationTemporality {
     package init(_ aggregationTemporaility: OTelAggregationTemporailty) {
```

Comment:
This type seems to have a typo too: `OTelAggregationTemporality`. The argument name also needs to be fixed.